libs/payments/cart/src/lib/manager.ts

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { v4 as uuidv4 } from 'uuid';
+import { Cart } from '../../../../shared/db/mysql/account/src';
+import { uuidTransformer } from '../../../../shared/db/mysql/core/src';
+import { Logger } from '../../../../shared/log/src';
+import { Cart as CartType } from '../gql';
+import { UpdateCartInput } from '../gql/update-cart.input';
+import { InvoiceFactory } from './factories';
+import { CartState, SetupCart } from './types';
+
+const DEFAULT_INTERVAL = 'monthly';
+
+export class CartManager {
+  private log: Logger;
+  constructor(log: Logger) {
+    this.log = log;
+  }
+
+  public async setupCart(input: SetupCart): Promise<CartType | null> {
+    const id = uuidv4({}, Buffer.alloc(16)).toString('hex');
```

Comment:
[nb] Would be nice for this to be in our own uuid util.

Reply:
Good call. I'll throw something into `libs/shared/db/mysql/core/src/lib/core`.